### PR TITLE
Add job offer

### PR DESCRIPTION
**Pull request by pierreburgy**
My PR is a:

💅 Enhancement

Main update on the:

Documentation

Add job offer in Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
-<p align="center"><img src="https://blog.strapi.io/content/images/2017/10/logo.png" width="318px" /></p>
+<p align="center"><strong>We're hiring!</strong> Located in Paris 🇫🇷 and dreaming of being full-time on Strapi? <a href="https://strapi.io/company#looking-for-talents">Join us</a>!</p>
```
[the remark this answers is not in the document]
Comment (Aurelsicoko) [reply README.md]: What do you mean by that?

**Comment by Aurelsicoko** [comment]
LGTM 👍

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kamalbennani) [reply README.md]: He's talking about the indentation :/

[the remark this answers is not in the document]
Comment (Aurelsicoko) [reply README.md]: Oh yes! I think @pierreburgy can improve this, not really important though 👍

[the remark this answers is not in the document]
Comment (pierreburgy) [reply README.md]: Good catch, I pushed an other commit. Thanks!

[the remark this answers is not in the document]
Comment (maturano) [reply README.md]: Thanks for understand me @kamalbennani 😄

**Pull request merged**